packages/p2p/src/components/order-details/__test__/order-details-complain-modal-radio-group.spec.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { fireEvent, render, screen } from '@testing-library/react';
+import OrderDetailsComplainModalRadioGroup from '../order-details-complain-modal-radio-group.jsx';
+
+describe('<OrderDetailsComplainModalRadioGroup/>', () => {
+    it('should render component with 3 radio buttons', () => {
+        render(<OrderDetailsComplainModalRadioGroup />);
+
+        expect(screen.getAllByRole('radio').length).toBe(3);
+    });
+
+    it('should select the checkbox when a radio button is selected', () => {
+        const mockFn = jest.fn();
+        render(<OrderDetailsComplainModalRadioGroup onCheckboxChange={mockFn} />);
```

Comment:
Same here, onCheckboxChange={jest.fn()} or onCheckboxChange={mockFn}

Reply:
I am checking if that mockFn is called in `Line 17`. Hence I am storing the ref in a variable

Reply:
Understand, no more worries, thx for the explanation!